website/docs/r/db_instance.html.markdown

```diff
@@ -1,14 +1,14 @@
 ---
-layout: "aws"
```

Comment:
Please note that we are going to revert the content of this documentation file to the current content (plus the addition of the new attribute) to prevent breaking open pull requests.